dart/lib/src/protocol/contexts.dart

```diff
@@ -25,6 +25,13 @@ class Contexts extends MapView<String, dynamic> {
           Gpu.type: gpu,
         });
 
+  factory Contexts.fromJson(Map<String, dynamic> data) => Contexts(
```

Comment:
missing runtimes, browser, and GPU, also we have consts for the contexts keys like `Device.type` instead of hardcoded Srings

Reply:
✅